www/js/metrics.jsx

```diff
@@ -1,27 +1,79 @@
 import React from "react";
 import { angularize } from './angular-react-helper';
-import { string ,object, array} from "prop-types";
+import { string ,object, array,any} from "prop-types";
 import { Card } from "react-native-paper";
 
 // Note the camelCase to dash-case conventions when translating to .html files!
 // val with explicit call toString() to resolve bool values not showing
-const metricsCard = ({ metricCardHeading, defaultSummaryVar}) => {
+
+
+
+const metricsCard = ({ mainMetrics, mainMetricsProp, summaryData}) => {
```

Comment:
Is this not a good way to pass in these variables? They seem to not behave like objects, even though they have properties? Is it specific to angular? Will keep looking into this. Putting them in console.log gives NaN and zero